src/dashboard/queue_distribute/queue_distribute.py

```diff
@@ -40,7 +40,7 @@
 def get_study_from_github(config):
     try:
         args = ["--depth", "1", config["url"], f"{BASE_DIR}/studies"]
-        if config["tag"]:
+        if config.get("tag"):
```

Comment:
this is an unrelated change i had hanging around locally from the last time i was working on message queues.

Reply:
Looks like a prime spot for a walrus operator...  :eyes: :smile: